toolset/utils/docker_helper.py

```diff
@@ -202,12 +202,18 @@ def watch_container(docker_container, docker_file):
             if hasattr(test, 'docker_cmd'):
                 docker_cmd = test.docker_cmd
 
+            # Expose ports in debugging mode
+            ports = {}
+            if self.benchmarker.config.mode == "debug":
+                ports = {test.port: test.port}
+
             container = self.server.containers.run(
                 "techempower/tfb.test.%s" % test.name,
                 name=name,
                 command=docker_cmd,
                 network=self.benchmarker.config.network,
                 network_mode=self.benchmarker.config.network_mode,
+                ports=ports,
```

Comment:
Ah yes, without this you would need to run a docker container on the `tfb` docker network in order to curl. Good catch.